cmd/admin-profile-stop.go

```diff
@@ -118,7 +119,12 @@ func mainAdminProfileStop(ctx *cli.Context) error {
 
 	fi, e := os.Stat(downloadPath)
 	if e == nil && !fi.IsDir() {
-		e = moveFile(downloadPath, downloadPath+"."+time.Now().Format("2006-01-02T15:04:05.999999-07:00"))
+		timeStampSeperator := ":"
+		if runtime.GOOS == "windows" {
+			timeStampSeperator = "-"
+		}
+		dtTmFormat := "2006-01-02T15" + timeStampSeperator + "04" + timeStampSeperator + "05.999999-07" + timeStampSeperator + "00"
```

Comment:
lets make this a global constant @BigUstad

Reply:
I actually meant to make the entire time stamp format itself as global and also perhaps use - for both windows and unix. It is annoying to have ":" even on Linux environment.